### PR TITLE
User to comments

### PR DESCRIPTION
**Pull request by heimrih**
User and comments

### CODE REVIEW

**Comment by heimrih** [comment]
ready

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -11,6 +13,7 @@ def new
 
   def create
     @comment = @post.comments.create comment_params
```
Comment (ngocvt-0484) [inline app/controllers/comments_controller.rb]:
```suggestion
    @comment = @post.comments.create comment_params.merge(user_id: current_user.id)
```

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -11,6 +13,7 @@ def new
...
+    @comment.user_id = current_user.id
```
Comment (ngocvt-0484) [inline app/controllers/comments_controller.rb]: so remove it

**Comment by heimrih** [comment]
ready

**Pull request merged**